vms/event/templates/event/create.html

```diff
@@ -44,7 +44,28 @@
                         </div>
                     </div>
                 {% endif %}
-				
+                {% if form.description.errors %}
```

Comment:
@anjali-dhanuka I was talking about this, since the template is modified this will be shown right? So we need to add a check for this at least in 1 test.